File: Assets/Scripts/PAKTOS.cs
using UnityEngine;
using System.Collections;
using UnityEngine.SceneManagement;

public class PAKTOS : MonoBehaviour {
   
    bool  isLoading;
    Transform[] hideStuff;
    Transform[] showStuff;

    void Start (){
        UnityEngine.Cursor.visible = false;
    }

    void Update (){
        if (Input.anyKey && !isLoading)
        {
            for (int hideIt = 0; hideIt < hideStuff.Length; hideIt++)
            {
                hideStuff[hideIt].gameObject.SetActive(false);
            }
            for (int showIt = 0; showIt < showStuff.Length; showIt++)
            {
                showStuff[showIt].gameObject.SetActive(true);
            }
            Load();

        
        }
    }

    void Load (){
        if (!isLoading)
        {
            isLoading = true;
            SceneManager.LoadSceneAsync("Test_Scene", LoadSceneMode.Single);
        }
   
    }
}

File: Assets/Scripts/PAKTOS.js
import UnityEngine.SceneManagement;

var isLoading : boolean;
var hideStuff : Transform[];
var showStuff : Transform[];

function Start ()
{
    UnityEngine.Cursor.visible = false;
}

function Update()
{
    if (Input.anyKey && !isLoading)
    {
        for (var hideIt = 0; hideIt < hideStuff.length; hideIt++)
        {
            hideStuff[hideIt].gameObject.SetActive(false);
        }
        for (var showIt = 0; showIt < showStuff.length; showIt++)
        {
            showStuff[showIt].gameObject.SetActive(true);
        }
        Load();

        
    }
}

function Load()
{
    if (!isLoading)
    {
        isLoading = true;
        SceneManager.LoadSceneAsync("Test_Scene", LoadSceneMode.Single);
    }
   
}